docs/source/evaluation.rst

```diff
@@ -71,20 +71,20 @@ How-To
 
 Given a serialized state of a fully optimized trajectory (here: ``optimized.pbstream`` file), the ground truth relations can be generated with the following command:
 
-.. code-block::
+.. code-block:: console
   
   cd <build>  # (directory where Cartographer's binaries are located)
   ./cartographer_autogenerate_ground_truth -pose_graph_filename optimized.pbstream -output_filename relations.pbstream -min_covered_distance 100 -outlier_threshold_meters 0.15 -outlier_threshold_radians 0.02
 
 Then, a non-optimized trajectory ``test.pbstream`` can be evaluated against the generated relations with:
 
-.. code-block::
+.. code-block:: console
 
   ./cartographer_compute_relations_metrics -relations_filename relations.pbstream -pose_graph_filename test.pbstream
 
 This will produce output in this form:
 
-.. code-block::
+.. code-block:: console
```

Comment:
This should be the language for syntax highlighting. Since the output isn't bash commands: `none`?

Reply:
Didn't know there's also `none`. Nevertheless, it behaves the same as `console` in my local build, so should I change it?